crates/forge/tests/e2e/common/runner.rs

```diff
@@ -48,3 +49,18 @@ pub(crate) fn setup_package(package_name: &str) -> TempDir {
 
     temp
 }
+
+pub(crate) fn gen_current_branch() -> String {
```

Comment:
I think some `///` comment here explaining what it does would be useful. It might not be clear for future devs that it is trying to use env var set by the github action.